package.json

```diff
@@ -35,7 +32,7 @@
     "react-apollo": "^2.1.9",
     "react-cookies": "^0.1.0",
     "react-dom": "^16.4.1",
-    "react-router-dom": "^4.3.1"
+    "react-router-dom": "next"
```

Comment:
why use an alpha/beta version of react-router-dom? It was already installed - did you need to re-add it?